### DISCUSSION TITLE
Video as background

### DISCUSSION THREAD

**Discussion by Stibo** [vidstack/player#1353] [Q&A] [answered]
I've found nothing about how to use Vidstack videos as background. Lets say we have a fullscreen mood section, and we want to play a video there, filling the whole screen. How can we disable/hide the controls completely? controls: false shows the Vidstack controls and controls:true shows the default provider controls. How can we easily get rid of all the controls and use the video as background without interaction? Are there other tips how to implement a video with Vidstack correctly if we want to use it as background?

**Comment by mihar-22** [comment, marked as answer]
Don't render the default layout and don't set `controls`. I'm contemplating adding a `hideControls` property to the default video layout to help with this.

**Reply by Stibo** [reply]
Sounds good, thanks! I think a `hideControls` would be neat.
